packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.template.ts

```diff
@@ -1,13 +1,7 @@
 import { ElementViewTemplate, html, ref, repeat, when } from '@microsoft/fast-element';
-import { createTabster, getGroupper, getMover } from 'tabster';
```

Comment:
what about donut chart and tests?

Reply:
In donut chart also, tabster will not work

Reply:
Ohh okay, will remove from there as well

Reply:
Not used in tests